pkgs/tools/games/slipstream/wrapper.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
I believe you should not specify a shebang. @drupol wdyt?

Reply:
Or use this `#! @shell@` and provide replace it with `substitute --subst-var-by shell stdenv.shell`

Reply:
I don't know. @pennae suggested to use `makeShellWrapper`.

Reply:
makeWrapper is almost the same as `makeShellWrapper`.

Reply:
I'm totally useless here, I don't know what's the best option to be honest :disappointed:

Reply:
> I'm totally useless here, I don't know what's the best option to be honest disappointed

No you aren't :hugs:. I feel like `wrapper.sh` is not needed and as you suggested it can be done with multiple `--run` options.

Reply:
Since we're removing the wrapper anyway I'll resolve this, but thanks for discussing it :)